src/srs-filter.c

```diff
@@ -38,8 +40,14 @@ static int threads = 0;
 static int CONFIG_verbose = 0;
 static int CONFIG_forward = 0;
 static int CONFIG_reverse = 0;
+static int CONFIG_reverse_null = 0;
 static char *CONFIG_socket = NULL;
+static char *CONFIG_user = NULL;
+static char *CONFIG_pidfile = NULL;
 static char **CONFIG_domains = NULL;
+static char *CONFIG_alias_cmd = NULL;
+static char *CONFIG_alias_regex_str = "^\\(.*\\)\\.\\.\\. deliverable";
```

Comment:
can you add a comment describing what this matches? I'm assuming the parentheses here are for string extraction? Like, "Regex to extract from the beginning of the line, up to a literal eclipses followed by the word deliverable".